pygeos/_geometry.pyx

```diff
@@ -75,3 +79,114 @@ def get_parts(object[:] array):
                 idx += 1
 
     return parts, index
+
+
+@cython.boundscheck(False)
+@cython.wraparound(False)
+def collections_1d(
+    object geometries,
+    object indices,
+    int geometry_type = 7,
+    int ndim = 2
```

Comment:
I don't see that `ndim` is ever passed in from the Python side?  Should this parameter be removed?